packages/app/src/cli/models/extensions/specification.ts

```diff
@@ -15,11 +15,25 @@ export type ExtensionFeature =
   | 'single_js_entry_path'
   | 'app_config'
 
+export type ConfigExtensionFeature = 'app_config'
+
+export type ExtensionSpecificationManagementExperience = 'cli' | 'app_config'
```

Comment:
I think this can just be `ManagementExperience` or `ExtensionManagementExperience`